Competition2018/src/main/java/competition/subsystems/lean/LeanSubsystem.java

```diff
@@ -1,36 +1,49 @@
 package competition.subsystems.lean;
 
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 
 import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
 import xbot.common.properties.DoubleProperty;
 import xbot.common.properties.XPropertyManager;
 
 @Singleton
 public class LeanSubsystem extends BaseSubsystem {
 
 	DoubleProperty leanSpeed;
+	CommonLibFactory clf;
+	double speed;
+	
+	public XCANTalon motor;
 	
 	@Inject
-	public LeanSubsystem(XPropertyManager propMan) {
-		propMan.createPersistentProperty("LeanSpeed", 0.2);
+	public LeanSubsystem(CommonLibFactory clf, XPropertyManager propMan) {
+		this.clf = clf;
+		leanSpeed = propMan.createPersistentProperty("LeanSpeed", 0.2);
+	}
+	
+	public void temporaryHack() {
+		motor = clf.createCANTalon(40);
 	}
 	
+	
 	/**
 	 * makes the climb arm lean left
 	 */
 	public void leanLeft() {
-		double speed = leanSpeed.get();
-
+		speed = leanSpeed.get();
+		motor.set(ControlMode.PercentOutput, speed);
```

Comment:
change this to ` motor.simpleSet(leanSpeed.get());`

Reply:
done